tests/functional/aws-node-sdk/test/object/initiateMPU.js

```diff
@@ -65,5 +66,118 @@ describe('Initiate MPU', () => {
                 done();
             });
         });
+
+        describe('with tag set', () => {
+            it('should be able to put object with 10 tags',
+            done => {
+                const taggingConfig = generateMultipleTagQuery(10);
+                s3.createMultipartUpload({
+                    Bucket: bucket,
+                    Key: key,
+                    Tagging: taggingConfig,
+                }, err => {
+                    assert.ifError(err);
+                    done();
+                });
+            });
+
+            it('should allow putting 50 tags', done => {
+                const taggingConfig = generateMultipleTagQuery(50);
+                s3.createMultipartUpload({
+                    Bucket: bucket,
+                    Key: key,
+                    Tagging: taggingConfig,
+                }, err => {
+                    assert.ifError(err);
+                    done();
+                });
+            });
+
+            it('should return BadRequest if putting more that 50 tags',
+            done => {
+                const taggingConfig = generateMultipleTagQuery(51);
+                s3.createMultipartUpload({
+                    Bucket: bucket,
+                    Key: key,
+                    Tagging: taggingConfig,
+                }, err => {
+                    assert(err, 'Expected err but did not find one');
```

Comment:
If assert.rejects doesn't work, I would remove this assert as it is already checked by the following assert.
This is similar to trying to assert `a` is 5 by first checking `a` is a number and then 5 when just asserting it is 5 would work.